app/views/pages/dfe_signin_help.html.erb

```diff
@@ -52,7 +51,7 @@
         </div>
         <div id="accordion-default-content-3" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-3">
             <p>Your approver will normally be the head or deputy head.<p>
-            <p>If they're not or you're not sure, don't worry. We just need to get some of your details so we can find them. Email our School Experience
+            <p>If they're not or you're not sure, you can email our School Experience
```

Comment:
@ekumachidi sorry this bit still sounded a bit odd so I've just amended it.

If your head or deputy head is not the approver, or you're not sure, you can email...

Reply:
@ekumachidi  not sure if you made this change as it still looks the same? But if not please can you update line 54 to say:
If your head or deputy head is not the approver, or you're not sure, you can email our school experience service team